recipes/uvicorn-worker/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "uvicorn-worker" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/uvicorn_worker-{{ version }}.tar.gz
+  sha256: f6894544391796be6eeed37d48cae9d7739e5a105f7e37061eccef2eac5a0295
+
+build:
+  skip: true  # [win]
+  skip: true  # [py<38]
```

You can make make this noarch b/c it won't be installable on Windows.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows. 

Before it would pass and be installable with OS-noarch, that would result in a broken package being available.


```suggestion
  skip: true  # [win]
  skip: true  # [py<38]
```